### DISCUSSION TITLE
`播放器控制栏背景色`  优化

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#3713] [功能建议 / Ideas]
不太确定是否真的算 bug，就先开个讨论吧

应该挺久了，`播放器控制栏背景色` 给播放器控制栏附上的这个黑色背景不对称，最右贴至边缘而最左没有（进度条是左右都没有），控制栏图标距上下的边距也不一致

![image](https://user-images.githubusercontent.com/49396421/193377403-3418978f-d23b-4a9a-8003-346d8906db98.png)

**Comment by the1812** [comment]
![image](https://user-images.githubusercontent.com/26504152/193379558-52695719-4198-42d0-824c-0d9cea29ed9c.jpg)
![image](https://user-images.githubusercontent.com/26504152/193379551-71340657-b9e4-4ea4-8253-f9d42f672eee.png)

**Reply by Geeyun-JY3** [reply]
怪了，我装个全新的脚本本体再就装这个组件仍能复现

**Reply by Geeyun-JY3** [reply]
似乎是水平偏移了
![image](https://user-images.githubusercontent.com/49396421/193394422-3ed0cee5-55ad-41a5-a426-b3894df5873f.png)

竖直方向暂时搞不清楚是什么情况

**Reply by Geeyun-JY3** [reply]
在 Edge 106.0.1370.42 (正式版本) (64 位) 测试也能测出水平偏移 bug：
![image](https://user-images.githubusercontent.com/49396421/195136736-866f0fff-3115-4446-8849-733d0ecd1134.png)

竖直偏移倒是没有。

**Reply by the1812** [reply]
Edge 上复现了

**Reply by Geeyun-JY3** [reply]
https://github.com/the1812/Bilibili-Evolved/discussions/3713#discussioncomment-3776920 在 Chrome 和百分浏览器都能复现，补充版本信息：
- Chrome：版本 106.0.5249.119（正式版本） （64 位）
- 百分浏览器：版本 5.0.1002.182（正式版本） （64 位） (Chromium 102.0.5005.167)

**Reply by Geeyun-JY3** [reply]
新版播放页面 Chrome 和百分浏览器都只能复现水平偏移（<https://github.com/the1812/Bilibili-Evolved/discussions/3713#discussioncomment-3861432>）了，https://github.com/the1812/Bilibili-Evolved/discussions/3713#discussioncomment-3776920 这种上下也有偏移的情况已无法复现。

**Reply by Tinhone** [reply]
Firefox V88.0 可复现 水平偏移 和 垂直偏移

**Reply by the1812** [reply]
我有点忘了, 原先开启 `播放器控制栏背景色` 是会填满左右空间的吗, 还是和原版一样有点间距

**Reply by Geeyun-JY3** [reply]
> 我有点忘了, 原先开启 `播放器控制栏背景色` 是会填满左右空间的吗, 还是和原版一样有点间距

我不记得了。
不过我觉得要是播放器控制栏有填充色后，左右空间还有点间距就不是很好看了。<https://github.com/the1812/Bilibili-Evolved/discussions/3713#discussioncomment-3776530> 这样左右空间无间距就挺好的。

**Reply by the1812** [reply]
那个截图里是还开了 `控制栏触摸优化`

**Reply by Geeyun-JY3** [reply]
> 我有点忘了, 原先开启 `播放器控制栏背景色` 是会填满左右空间的吗, 还是和原版一样有点间距

搜了一下 issue，至少 v1 是有间距的：https://github.com/the1812/Bilibili-Evolved/issues/363#issuecomment-595680422
没记错 v1 的文档当时也有这功能的截图，还能找到吗？

**Reply by the1812** [reply]
那应该是有间距的

**Reply by Geeyun-JY3** [reply]
利用 https://github.com/the1812/Bilibili-Evolved/discussions/4206#discussioncomment-5938148 提到的脚本返回旧版播放页后可稳定复现 <https://github.com/the1812/Bilibili-Evolved/discussions/3713#discussioncomment-3776920>，可验证新版播放页无上下偏移，旧版播放页两个方向的偏移都有。
这么说上下偏移 bug 和旧版播放页有关系。

**Comment by Geeyun-JY3** [comment]
旧版番剧播放页还是有上下偏移（准确的说是进度条下方空出了一部分）：
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/b5c66b5f-cee2-423d-af3d-d40f98deb8f2)

旧版视频播放页（旧版视频播放页还适配吗？）也是类似的：
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/6247c2b0-b9a0-4e27-b2e1-ead07bfb6d6e)

v2.7.2-53-g97c6184e1 和 v2.8.0-preview-77-g7e99bde31 均可复现。
